daprdocs/content/en/reference/environment/_index.md

```diff
@@ -8,12 +8,13 @@ weight: 300
-| DAPR_TOKEN_API                   | Your application | The token used for Dapr API authentication for requests from the application. Read [enable API token authentication in Dapr]({{< ref api-token >}}) for more information.
-| APP_TOKEN_API                    | Your application | The token used by the app to authenticate requests from Dapr. Read [authenticate requests from Dapr using token authentication]({{< ref app-api-token >}}) for more information.
-| DAPR_PLACEMENT_HOST              |Your application | The address for the Dapr Placement service. Only needed in self-hosted mode if you run your app (which creates actors) and you want to tell the app the location of the Placement service. This is never needed outside of local machine development.
-| DAPR_NETWORK                     | Dapr CLI         | Optionally used by the Dapr CLI to specify the Docker network on which to deploy the Dapr runtime.
-| NAMESPACE                        | Dapr runtime     | Used to specify a component's [namespace in self-hosted mode]({{< ref component-scopes >}})
+| Environment Variable | Used By                        | Description                                                                                                                                                                                                                                                                                                                    |
+| -------------------- | ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| APP_ID               | Your application               | The id for your application, used for service discovery                                                                                                                                                                                                                                                                        |
+| APP_PORT             | Your application               | The port your application is listening on                                                                                                                                                                                                                                                                                      |
+| APP_API_TOKEN        | Dapr sidecar, your application | The token used by the application to authenticate requests from Dapr API. Read [authenticate requests from Dapr using token authentication]({{< ref app-api-token >}}) for more information.                                                                                                                                   |
```

Comment:
One small comment came up when looking at this again and it best to change APP_API_TOKEN to just say "Your Application" since this is the only real place this env variable is used. @anhldbk - can you do this small change

| APP_API_TOKEN | Dapr sidecar, your application  --> Your application